presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -237,6 +237,12 @@ Property Name                                           Description
 
 ``iceberg.pushdown-filter-enabled``                     Experimental: Enable filter pushdown for Iceberg. This is     ``false``
                                                         only supported with Native Worker.
+
+``iceberg.rows-for-metadata-optimization-threshold``    The max partitions number to utilize metadata optimization.   ``1000``
```

Comment:
```suggestion
``iceberg.rows-for-metadata-optimization-threshold``    The maximum number of partitions in an Iceberg table to   ``1000``
```
I'm suggesting this revision of the definition for clarity, and (in the last sentence) a change to imperative for readability. I tested this revision in a local build, but if I didn't transfer my local edits to here correctly, I'm including a screenshot of my local doc build to show the suggestion clearly. 
<img width="825" alt="Screenshot 2024-04-01 at 2 01 39 PM" src="https://github.com/prestodb/presto/assets/7013443/a7df44a7-6d27-4740-be7e-85fdd9b9c8db">